### ISSUE TITLE
Question about real-time teleoperation deployment 

### ISSUE THREAD

**Issue by FinnJob** [LeCAR-Lab/human2humanoid#13]
Thank your for your excellent work!
I'd like to know how you achieve real-time remote control of a humanoid robot through an RGB camera. To my knowledge, during the play process, the input to the trained sim-to-real imitation policy is processed AMASS motion data. So, how can I obtain such processed human joint data in real time?
Thank you!

**Comment by TairanHe** [comment]
Hi, we use Apple Vision Pro and RGB camera (+HybrID) to get global positions of human bodies.

**Comment by TairanHe** [comment]
You could check

**Comment by FinnJob** [comment]
Thank you for your reply!
What I really want to know is how to use the joint data obtained from the RGB camera (+HybrID) as reference data for the robot after it has been converted into SMPL format. This is because I noticed in the code that during the data preprocessing stage, the SMPL joint data is converted into reference data for H1 using gradient descent.

**Comment by TairanHe** [comment]
The reason we could do this is because we are tracking sparse inputs instead of whole-body motions.

**Comment by FinnJob** [comment]
@TairanHe I get it. Thanks!

**Comment by yixxuan-li** [comment]
@FinnJob Can I have your email or WeChat? I want to ask for your expertise in deployment.

**Comment by FinnJob** [comment]
@TairanHe Hi, I've noticed that the joint positions obtained from the RGB camera (+HybrID) can not be directly used as reference targets in Isaac Gym, as there seems to be a matrix transformation between them. However, when using training data from the AMASS dataset, specifically from the file 'amass_copycat_take6_train.pkl', the joint positions can be directly used as targets without any matrix transformation. 

So, I'm wondering, when using joint positions obtained from the RGB camera (+HybrID) , are they subjected to some matrix transformation before being input into Isaac Gym?

**Comment by YechenZhou** [comment]
> @TairanHe Hi, I've noticed that the joint positions obtained from the RGB camera (+HybrID) can not be directly used as reference targets in Isaac Gym, as there seems to be a matrix transformation between them. However, when using training data from the AMASS dataset, specifically from the file 'amass_copycat_take6_train.pkl', the joint positions can be directly used as targets without any matrix transformation.
> 
> So, I'm wondering, when using joint positions obtained from the RGB camera (+HybrID) , are they subjected to some matrix transformation before being input into Isaac Gym?

@FinnJob Hi, could you tell me where I can obtain the file 'amass_copycat_take6_train.pkl‘？ I could not find it in PHC repo.

**Comment by FinnJob** [comment]
@YechenZhou Hi, you could follow the instructions in "Data Processing AMASS" of PHC repo.

**Comment by YechenZhou** [comment]
I solved it. Thanks! @FinnJob
